### ISSUE TITLE
When Organisation is changed to other value than Mendix, preview in Mendix fails

### ISSUE THREAD

**Issue by frederikbeun** [mendix/pluggable-widgets-generator#15]
When Using the MX Widget I changed the Organisation name to the name of our company.
Everything wents well until when running the new widget locally in mendix. Then we are getting a 404 - file not found for file: widgets/**mendix**/textbox/TextBox.js
As you see the path is here fixed mendix which didn't exist for us
After changing the package.json node "packagePath": "mendix", and rebuilding everything it worked.

**Comment by diego-antonelli** [comment]
Hi, thanks for your feedback. Are you using the version 0.1.5 published yesterday (25/07) ? Because we fixed a problem with the paths.

**Comment by frederikbeun** [comment]
I used version 0.1.4.
Br
Frederik

Van: Diego Antonelli <notifications@github.com>
Verzonden: vrijdag 26 juli 2019 9:45
Aan: mendix/pluggable-widgets-generator <pluggable-widgets-generator@noreply.github.com>
CC: Frederik Beun <Frederik.Beun@CLAREBOUT.COM>; Author <author@noreply.github.com>
Onderwerp: Re: [mendix/pluggable-widgets-generator] When Organisation is changed to other value than Mendix, preview in Mendix fails (#15)


Hi, thanks for your feedback. Are you using the version 0.1.5 published yesterday (25/07) ? Because we fixed a problem with the paths.
